packages/connection-form/src/components/connection-form-actions.tsx

```diff
@@ -118,4 +118,71 @@ function ConnectFormActions({
   );
 }
 
+export type ConnectionFormModalActionsProps = {
+  errors: ConnectionFormError[];
+  warnings: ConnectionFormWarning[];
+
+  onCancel(): void;
+  onSave(): void;
+  onConnect(): void;
+};
+
+export function ConnectionFormModalActions({
+  errors,
+  warnings,
+  onCancel,
+  onSave,
+  onConnect,
+}: ConnectionFormModalActionsProps): React.ReactElement {
+  return (
+    <div className={cx(formActionStyles)}>
+      {warnings.length > 0 && (
+        <div className={formActionItemStyles}>
+          <WarningSummary
+            data-testid="connection-warnings-summary"
+            warnings={warnings.map((warning) => warning.message)}
+          />
+        </div>
+      )}
+      {errors.length > 0 && (
+        <div className={formActionItemStyles}>
+          <ErrorSummary
+            data-testid="connection-error-summary"
+            errors={errors.map((error) => error.message)}
+          />
+        </div>
+      )}
+      <div className={cx(formActionItemStyles, formActionButtonsStyles)}>
+        <Button
+          data-testid="save-connection-button"
+          variant={ButtonVariant.Default}
+          disabled={false}
```

Comment:
If this value never changes, you don't really need to pass it. Or is it supposed to be derived from errors / warning?